antenna-workflow-steps/processors/enricher/antenna-sw360-enricher/src/main/java/org/eclipse/sw360/antenna/workflow/processors/SW360Enricher.java

```diff
@@ -63,7 +64,10 @@ public void configure(Map<String, String> configMap) throws AntennaConfiguration
         String sw360AuthServerUrl = getConfigValue(AUTH_SERVER_URL_KEY, configMap);
         String sw360User = getConfigValue(USERNAME_KEY, configMap);
         String sw360Password = getConfigValue(PASSWORD_KEY, configMap);
-        connector = new SW360MetaDataReceiver(sw360RestServerUrl, sw360AuthServerUrl, sw360User, sw360Password);
+        boolean sw360ProxyUse = Boolean.parseBoolean(getConfigValue(PROXY_USE, configMap));
```

Comment:
What is the default value here, if `PROXY_USE` is missing? (related to https://github.com/eclipse/antenna/pull/156/files#r281933181)

Reply:
```suggestion
        boolean sw360ProxyUse = getConfigValue(PROXY_USE, configMap) != null ? Boolean.parseBoolean(getConfigValue(PROXY_USE, configMap)) : false;
```
something like this?

Reply:
I would prefere:
```suggestion
        boolean sw360ProxyUse = Boolean.parseBoolean(getConfigValue(PROXY_USE, configMap, "false"));
```